static/src/javascripts/projects/common/modules/analytics/shouldCaptureMetrics.ts

```diff
@@ -1,9 +1,14 @@
 import type { ABTest } from '@guardian/ab-core';
 import { isInABTestSynchronous } from '../experiments/ab';
 import { integrateCriteo } from '../experiments/tests/integrate-criteo';
+import { integrateSmart } from '../experiments/tests/integrate-smart';
 import { prebidTimeout } from '../experiments/tests/prebid-timeout';
 
-const defaultClientSideTests: ABTest[] = [prebidTimeout, integrateCriteo];
+const defaultClientSideTests: ABTest[] = [
+	prebidTimeout,
+	integrateCriteo,
+	integrateSmart,
+];
```

Comment:
I think in this case we would want to force the linter to always keep this array over multiple lines.

```suggestion
const defaultClientSideTests: ABTest[] = [
	/* linter, please keep this array multi-line */
	prebidTimeout,
	integrateCriteo,
	integrateSmart,
];
```

Reply:
What's the advantage of doing this? 🤔

Reply:
Diffing becomes easier to parse. The linter will never collapse the array to a single line. This way, it’s also easier to navigate the blame or have useful inline commit info.

Technically, we only add and remove tests from this array, line 7 and 11 shouldn’t be touched by this PR.